Update Global Environment Variables (#1586)
astronomer/docs: astro/runtime-image-architecture.md

Each Runtime version in a given minor series supports only a single version of Apache Airflow. For specific version compatibility information, see [Runtime release notes](runtime-release-notes.md).
:::

## Default environment variables

> The following table list the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set by Astronomer, see [Global environment variables](platform-variables.md).

Reviewer: ```suggestion
The following table lists the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set on the Astro data plane, see [Global environment variables](platform-variables.md).
```

> The following table lists the default Runtime environment variables. You can change the default settings of these environment variables to meet the unique requirements of your organization. For information about the global environment variables set on the Astro data plane, see [Global environment variables](platform-variables.md).

Reviewer 2: Link to env vars doc as well/modify sentence to say, "To override..."

Author: Added the text and link.

| Environment Variable                       | Description                                                                                                          | Value                                   |
| ------------------------------------------ | -------------------------------------------------------------------------------------------------------------------- | --------------------------------------- |

> | `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT`                        | The time in seconds that queued Celery tasks are assumed to have stalled before they are automatically rescheduled. The default is zero.                                                                 | `integer`                |

Reviewer: ```suggestion
| `AIRFLOW__CELERY__STALLED_TASK_TIMEOUT`                        | The time in seconds that queued Celery tasks are assumed to have stalled before they are automatically rescheduled.  | `0`   |
```

| `AIRFLOW_CORE_PARALLELISM`                    | The maximum number of tasks that can run at the same time in a single Airflow environment.                                                               | `32`           |

> 

Reviewer 2: Extra space

Author: Fixed

> | `AIRFLOW_CORE_PARALLELISM`                    | The maximum number of tasks that can run at the same time in a single Airflow environment. The default is 32.                                                              | `string`            |

Reviewer: ```suggestion
| `AIRFLOW_CORE_PARALLELISM`                    | The maximum number of tasks that can run at the same time in a single Airflow environment.                                                               | `32`           |
```

Reviewer: I don't think the value types are important here, so I replaced them with the default values which seems to be the core piece of information this table should show. Value types can be inferred from the default values themselves, at least in the case of these integer values.
